cli/tests/helm-tests/helm_test.go

```diff
@@ -0,0 +1,70 @@
+package helm_tests
```

Comment:
Glad to see these tests. 1 comment: can we test for more than just lack of errors? Like verify something more than just err!=nil